app/controllers/LogIn.scala

```diff
@@ -0,0 +1,32 @@
+package controllers
+
+import com.gu.googleauth.{GoogleAuthConfig, LoginSupport}
+import play.api.libs.ws.WSClient
+import play.api.mvc.{BaseController, Call, ControllerComponents}
+
+import scala.concurrent.ExecutionContext
+
+// based on https://github.com/guardian/play-googleauth/blob/master/example/app/controllers/Login.scala
+class LogIn(
```

Comment:
Small point - should be `class Login`

Reply:
'Log in' is two words, isn't it? A noun phrase and a verb phrase.  There's no such thing as a login!

Reply:
It's widely used across the internet in this context as `login` no? https://www.google.co.uk/search?q=login&oq=login&aqs=chrome..69i57j69i60l4j0.1036j0j4&sourceid=chrome&ie=UTF-8

Reply:
In that case I think it could be either, so I'll merge it as-is but feel free to change it later.